training/distributed_training/pytorch/model_parallel/GPT2_huggingface_trainer/submit_smp_trainer.ipynb

```diff
@@ -0,0 +1,623 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    if model_config == "gpt2-small":
</pre><p>Please make sure all config are tested.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3577/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Yes, all the configurations are tested and working!